README.md

```diff
@@ -21,118 +21,10 @@ Ketch is an application delivery framework that facilitates the deployment and m
 ## Architecture Diagram
 ![Architecture](./img/ketch-architecture.png)
 
-## Getting Started
-
-### Download and Install Ketch
-The latest Ketch release can be found [here](https://github.com/shipa-corp/ketch/releases). Use the following commands
-to install Ketch,
-
-Install latest at /usr/local/bin default location
-
-```bash
-curl -s https://raw.githubusercontent.com/shipa-corp/ketch/main/install.sh | bash
-```
-
-Alternatively, you can install specific tag at a target location, for example command below installs ketch version v0.4.0 in current directory:
```

Comment:
I _think_ this bit about version and install dir are missing from theketch.io docs. They be useful.

Reply:
good catch @stinkyfingers, fixed, https://learn.theketch.io/docs/getting-started#ketch-cli